associations.rb

```diff
@@ -83,7 +95,7 @@ def random_loc; (('a'..'e').to_a.sample) + rand(1..5).to_s; end
 
 user = User.create!(name: "John Smith")
 room = hotel.rooms.first
-b = Booking.create!(guest: user, room: room, check_in: Time.now)
+b = Booking.create!(guest: user.id, room: room.id, check_in: Time.now, hotel: hotel)
```

Comment:
You shouldn't need to store a hotel in a booking - you're already storing a room which by definition has a hotel attached.